frontend/packages/metal3-plugin/src/feature.ts

```diff
@@ -0,0 +1,44 @@
+import { k8sGet, K8sResourceKind } from '@console/internal/module/k8s';
```

Comment:
I'd suggest naming this file `features.ts` (plural) to establish a convention, similar to `src/models` or `src/redux`.